packages/ember-metal/lib/computed_macros.js

```diff
@@ -648,11 +648,11 @@ export function readOnly(dependentKey) {
 export function deprecatingAlias(dependentKey) {
   return computed(dependentKey, {
     get(key) {
-      Ember.deprecate(`Usage of \`${key}\` is deprecated, use \`${dependentKey}\` instead.`);
+      Ember.deprecate(`Usage of \`${key}\` is deprecated, use \`${dependentKey}\` instead.`, false, { id: 'ember-metal.deprecating-alias', until: '3.0.0' });
```

Comment:
Yep, sounds exactly correct. Then all the calling sites need to be updated to pass in their own options arg.